proxy-robots-module/src/main/java/orwell/proxy/robot/ColourSensor.java

```diff
@@ -85,4 +81,18 @@ public void clear() {
     public void accept(final IRobotElementVisitor visitor) {
         visitor.visit(this);
```

Comment:
Ok then, I agree it is not good consistency wise.
I'll remove the null check for the visitor (which anyway won't fail since it will fall in the DEFAULT-UNDEFINED case which is handled)